apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -365,6 +366,7 @@ def evaluate_with_source(
     # for exponentiation. In Python, only the latter can be used.
     # Also replace the unicode minus with the normal one.
     expr = expr.replace("^", "**").replace("\u2212", "-")
```

Comment:
```suggestion
    expr = expr.replace("^", "**")
    # Also replace the unicode symbols with ASCII ones.
```
The Unicode minus is already converted by unidecode, no need to do it separately.

Reply:
Had to revert this, tests fail unless this replacement is used.